manifests/s/Sass/Dart-Sass/1.83.4/Sass.Dart-Sass.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: Sass.Dart-Sass
+PackageVersion: 1.83.4
+PackageLocale: en-US
+Publisher: Sass
+PublisherUrl: https://github.com/sass
+PublisherSupportUrl: https://github.com/sass/dart-sass/issues
+PackageName: Dart-Sass
+PackageUrl: https://github.com/sass/dart-sass
+License: MIT

```

```suggestion
License: MIT
LicenseUrl: https://github.com/sass/dart-sass/blob/main/LICENSE
Copyright: Copyright (c) 2016, Google Inc.
CopyrightUrl: https://github.com/sass/dart-sass/blob/main/LICENSE
```

They can be extracted from the official repository.